README.md

```diff
@@ -14,9 +14,7 @@ Thanks for your kind support and appreciation for this project.
 This was developed on a Mac, I am not sure how to configure on a PC. Sorry!
 
 - Node: `brew install node`
-- SASS: `brew install sass`
-- Grunt CLI: `npm install -g grunt-cli`
-- Bower: `npm install -g bower`
+- Webpack: `npm install -g webpack webpack-dev-server`
```

Comment:
You actually should not need to install these dependencies globally - `npm start` will scope the cli commands to the installed modules in `node_modules`.

But, if you don't mind giving it a shot before editing this, I'd appreciate it!